src/database/migrations/20191206111012-create-call.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("Calls", {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true, 
+        autoIncrement: true
+      },
+      type_of_contact: {
+          type: Sequelize.STRING,
+          allowNull: false
+      },
+      date: {
+          type: 'TIMESTAMP',
+          defaultValue: new Date('03/25/2015'),
+          allowNull: false
+      },
+      day: {
+          type: Sequelize.STRING,
+          allowNull: false
+      },
+      time: {
+          type: Sequelize.STRING,
+          allowNull: false
+      },
+      call_duration: {
+          type: Sequelize.STRING,
+          allowNull: false
+      },
+      type_of_call_id: {
+          type: Sequelize.INTEGER,
+          references: {
+            // This is a reference to another model
+            model: 'Types_of_calls',
+            // This is the column name of the referenced model
+            key: 'id'
+            },
+          allowNull: false
+      },
+      volonteer_id: {
+          type: Sequelize.INTEGER,
+          references: {
+            // This is a reference to another model
+            model: 'Volunteers',
+            // This is the column name of the referenced model
+            key: 'id'
+            },
+          allowNull: false
+      },
+      caller_id: {
+          type: Sequelize.INTEGER,
+          references: {
+
+          // This is a reference to another model
+          model: 'Callers',
+      
+          // This is the column name of the referenced model
+          key: 'id',
+          },
+      allowNull: false
+      },
+      call_description: {
+        type: Sequelize.INTEGER,
+        references: {
+
+        // This is a reference to another model
+        model: 'Call_descriptions',
+    
+        // This is the column name of the referenced model
+        key: 'id',
+        },
+        allowNull: true
+    },
+      created_at: {
+        type: 'TIMESTAMP',
+        defaultValue: new Date('03/25/2015'),
+        allowNull: false
+      }
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable("Calls");
+  }
+};
```

Comment:
Define everything in a single migration. I can explain why this is important during standup.

Reply:
It means that migrations for all tables need to put in one migrations file?

Reply:
Yes.